src/OrchardCore/OrchardCore.Email.Abstractions/Extensions/EmailServiceExtensions.cs

```diff
@@ -11,20 +11,44 @@ public static class EmailServiceExtensions
     /// <param name="emailService">The <see cref="IEmailService"/>.</param>
     /// <param name="to">The email recipients.</param>
     /// <param name="subject">The email subject.</param>
-    /// <param name="body">The email body.</param>
-    /// <param name="isHtmlBody">Whether the <paramref name="body"/> is in HTML format or not. Defaults to <c>true</c>.</param>
-    /// <returns></returns>
+    /// <param name="htmlBody">The email body in HTML format.</param>
```

Comment:
The comments should say the values are optional (can be null). Maybe use the `?` operator to be specific.